test/testQueryClient.ts

```diff
@@ -96,5 +97,10 @@ export const queryClientWithMockData = (): QueryClient => {
     () => behandlereDialogmeldingMock
   );
 
+  queryClient.setQueryData(
+    behandlerdialogQueryKeys.behandlerdialog(ARBEIDSTAKER_DEFAULT.personIdent),
+    () => []
+  );
```

Comment:
Jeg og JML var litt usikker på om dette er nødvendig på en post-mutation, eller om det er viktigst for`get`?

Reply:
Den brukes jo bare til å sette cachen for GET-kallet så har ikke noe å si for POST direkte. Men man gjør gjerne en invalidering av cachen (som fører til ny GET) ved en POST-mutation så hvis man vil teste resultatet av den har det jo noe å si.